webapp/src/utils/lyraConfig.ts

```diff
@@ -63,9 +64,10 @@ export class LyraConfig {
             KIND_BY_FORMAT_VALUE[project.messages.format],
             project.messages.path,
             project.translations.path,
+            project.languages ?? ['en'], // default language to be english if not provided
```

Comment:
Will a target project with only one language ever be useful?

Reply:
no, also there is no good reason to assume en is a default language, but I think a developer at least will start with en.yaml file

Reply:
For projects with type script messages, en is supposed to be generated from the messages, see https://github.com/zetkin/app.zetkin.org/blob/main/CONTRIBUTING.md#how-it-works

Reply:
For projects with yaml messages instead of typescript messages, `en` is the source of the messages.

In conclusion, I don't think there is a use case for editing `en` at all.

For this PR, let's require languages to be configured in `lyra.yml` instead of it being optional. We'll create a separate issue for denying edits to `en`.

Reply:
After discussing it, we decided that app.zetking.org is the special case, where `en` cannot be updated, and that `['en']` is fine as a default.